infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -117,15 +118,15 @@ public StreamLogFiles(ServerContext serverContext, boolean noVerify) {
         channelsToSync = new HashSet<>();
         this.verify = !noVerify;
         this.dataStore = StreamLogDataStore.builder().dataStore(serverContext.getDataStore()).build();
-        String logSizeQuotaParam = (String) serverContext.getServerConfig().get("--log-size-quota-bytes");
+        String logSizeLimitPercParam = (String) serverContext.getServerConfig().get("--log-size-quota-percentage");
 
-        logSizeLimit = Long.parseLong(logSizeQuotaParam);
+        logSizeLimitPercentage = Double.parseDouble(logSizeLimitPercParam);
```

Comment:
Is is better to make `logSizeLimitPercentage` a local parameter and pass it into initStreamLogDirectory() ?

Reply:
done.